### DISCUSSION TITLE
Creating Custom Templates

### DISCUSSION THREAD

**Discussion by malsharman** [Budibase/budibase#3537] [Help]
Hi

Budibase ships with templates, is there a way to create my own templates for apps or forms? Happy to hack away on config files but wondering what the starting points would be to work it out.

**Comment by mjashanks** [comment]
at the moment, there's no way for you to list your templates in the "Create App". However, you can export your app, then import it. 

Not a great experience, but it's exactly the same thing technically.

We will look to address this at some point - don't know when!
